JASP-Engine/JASP/R/correlationbayesian.R

```diff
@@ -15,2850 +15,1312 @@
+        #
+        if (m > 1 & options[["reportN"]]) {
+          if (nVariables >= 2) {
+            allItemInfo[["n"]] <- NULL
+          }
+          # Note(Alexander): After the first time we added "n", remove it from the other
+          # 
+          itemNames <- setdiff(itemNames, "n")
         }
+        
+        rowItemNames <- .bCorRowNames(options, itemNames, "method"=methodName)
+        
+        for (k in seq_along(itemNames)) {
+          if (k==1 & m==1) {
+            # First item gets the variable name
+            tempRow <- list("variable"=var1, "itemColumn"=rowItemNames[[k]])
+          } else {
+            tempRow <- list("variable"="", "itemColumn"=rowItemNames[[k]])
+          }
+          
+          if (nVariables >= 2) {
+            itemInfo <- as.list(allItemInfo[[k]])
+          } else {
+            itemInfo <- as.list(emptyCellInfo)
+          }
+          names(itemInfo) <- var2Names
+          tempRow <- modifyList(tempRow, itemInfo)
+          table$addRows(tempRow, rowNames=paste0(var1, methodName, itemNames[k]))
+        } # End items loop
+      } # End loop over the methodItems
+    } # End var1 loop
+  } # End matrix fill
+  
+  for (i in seq_along(errorFootnotes)) 
+    table$addFootnote(message = errorFootnotes[[i]], rowNames = errorRowName[[i]], colNames = errorColName[[i]])
+  
+  # Note(Alexander): Now add footnotes
+  #
+  if (options[["flagSupported"]]) {
+    for (i in seq_along(supportFootnotes)) {
+      table$addFootnote(message=supportFootnotes[[i]], symbol = supportSymbols[[i]], rowNames = supportRowName[[i]], colNames = supportColName[[i]])
     }
-    return(result)
+  }
+  return(allBfObjects)
 }
 
-# 2.4 The Marsman MH sampler
-
-# .logTarget <- function(z, n, r, kappa) {
-# 	# z is Fisher's transformation for r, but also use it for rho
-# 	# The Fisher z transform and the log (likelihood*prior*Jacobian) of the tranformation
-# 	(0.5*(n - 1))*log(1 - tanh(z)^2) - (n - 1 - 0.5)*log(1 - tanh(z)*r)+log(1-tanh(z)^2)/kappa
-# }
-
-.logTarget <- function(rho, n, r, kappa=1) {
-  # More correctly, with rho=rhoProp, this is upper term of the acceptance ratio. 
-  # Recall that in the numerator and denominator of the acceptance ratio are given by
-  # 
-  #   likelihood(rhoProp)*prior(rhoProp)*proposal(rhoCurrent | rhoProp)
-  #   likelihood(rhoCurrent)*prior(rhoCurrent)*proposal(rhoProp | rhoCurrent)
-  # 
-  # respectively, Christian Robert (2015). As the proposal is defined on Fisher transform atanh(rho), 
-  # we have a Jocobian. The Jacobian in the upper part is (1-rhoCurrent^2)^(-1) and in the lower part
-  # is (1-rhoProp^2)^(-1).
-  # Note that since prior(rhoProp) \propto (1-rhoProp^2)^(alpha-1), we can drop the one due 
-  # to the Jacobian term of the denominator ending up in the numerator.
-  # 
-  # For the likelihood we use Jeffreys's approximation 
-  #   rho \propto (1-rho^2)^((n-1)/2)*(1-rho*r)^((3-2*n)/2)
-  # 
-  # The log prior is (alpha-1)*log(1-rho^2) # where 
```

Comment:
Noticed quite a few of these (`&`) -- considering you don't want to do elementwise comparisons, use `&&`.
Same for `|` in your if-statements.